src/MarketingPageAcceptanceTests.AuthorityTests/Features/Capabilities/DispalySolutionsCapabilitysEpics.feature

```diff
@@ -14,4 +14,10 @@ Scenario: Solution Capability Epics - Must Epics and May Epics
 Scenario: Solution Capability Epics - Epic Met or Not Met
 	Given that Epic Pass or Fail Statuses have been provided for each Epic
 	When a User previews the Marketing Page
-	Then the Epic pass or fail status is visible
\ No newline at end of file
+	Then the Epic pass or fail status is visible
+
+# Need to figure out what happens here, but have used existing steps on the assumption that a validation message will be displayed on save
+Scenario: Solution Capability Epics - Epic provided for Capability not provided
+	Given that Epics are provided for Capabilities not provided
+	When the User attempts to save
+	Then the User is provided with a validation error message
```

Comment:
I can't see a pending step for this

Reply:
That's an existing step that's used elsewhere